cmd/airflow.go

```diff
@@ -226,10 +226,18 @@ func airflowInit(cmd *cobra.Command, args []string) error {
 		projectName = strings.Replace(strcase.ToSnake(projectDirectory), "_", "-", -1)
 	}
 
-	acceptableAirflowVersions := []string{"1.9.0", "1.10.5"}
+	r := houston.Request{
+		Query:     houston.DeploymentConfigRequest,
+	}
+
+	wsResp, err := r.DoWithClient(client)
+	if err != nil {
+		return err
```

Comment:
@schnie FYI ^

Reply:
```
$ astro dev init
Error: unable to ask Houston API for airflow versions: HTTP DO Failed: Post http://localhost:8871/v1: dial tcp [::1]:8871: connect: connection refused
Usage:
 
```

Reply:
or
```
Error: unable to ask Houston API for airflow versions: You do not have the appropriate permissions for that: Your token has expired. Please log in again.
```
@greg sounds good?

Reply:
@ashb @andriisoldatenko yep, if you are not connected to a cluster this would fail so definitely need to handle it. That was part of the thinking that I quickly outlined here: https://github.com/astronomer/issues/issues/596#issuecomment-558767703.

Essentially, run the query, use the response if possible, otherwise fallback to `astronomerinc/ap-airflow:latest-onbuild`, which will always point to our default image. Currently `1.10.3-alpine3.10`, but could be swapped for `buster` variant when we feel good about it.

Reply:
If we can't connect, maybe we say `Unable to query for supported Airflow versions. Falling back to unversioned Dockerfile tag. See documentation for more details`.